.github/workflows/test.yml

```diff
@@ -52,12 +50,18 @@ jobs:
         poetry env use "${{ matrix.python-version }}"
         poetry install
 
-    - name: Run pytest
+    - name: Run pytest linux
+      if: (matrix.os != 'windows-latest')
       run: |
         poetry run coverage run --parallel -m pytest -m "$PYTEST_MARKERS"
+    
+    - name: Run pytest windows
+      if: (matrix.os == 'windows-latest')
+      run: |
+        poetry run coverage run --parallel -m pytest -m $Env:PYTEST_MARKERS
```

Comment:
@visch Since the pytest markers string doesn't really need to be an env var, wdyt of adding `pytest_markers: ["not concurrent"]` to the matrix and using a single step with

```yaml
    - name: Run pytest
      run: |
        poetry run coverage run --parallel -m pytest -m "${{ matrix.pytest_markers }}"
```

Reply:
Fixed, did it with Env variables looks like this is how do you "expansion" with github actions.

Reply:
@visch Ah, even better. Thanks!